docs/faq/autoscaling.md

```diff
@@ -0,0 +1,45 @@
+
+# What is autoscaling?
+
+Autoscaling refers to a system where the resources allocated to an application
+can be dynamically adjusted in response to load on the system.
+
+In Fleet, autoscaling is applied to application server instances (Frontend
+instances). This is designed to allow an application to scale across a
+reasonable range of traffic levels. There will eventually be scalability
```

Comment:
"eventually" sounds a bit strange. Kind of sounds like "scalability limits" might be a feature we're adding later.

"However the backend database can not autoscale, so an appropriate Fleet size should be chosen to handle your peak load's database requirements." or something.